language/move-core/types/Cargo.toml

```
@@ -25,6 +26,7 @@ ethnum = { version = "1.4", default-features = false }
 hashbrown = { version = "0.14", default-features = false, features = ["ahash"] }
 bcs = { default-features = false, git = "https://github.com/eigerco/bcs.git", branch = "master" }
 arbitrary = { version = "1.3", default-features = false, features = ["derive_arbitrary"], optional = true }
+parity-scale-codec = { version = "3.6.9", default-features = false, features = ["derive"] }
```

nit suggestion for easier maintenance of up-to-date package versions in the TOML 
```suggestion
parity-scale-codec = { version = "3.6", default-features = false, features = ["derive"] }
```